browser/flagr-ui/src/components/Flags.vue

```diff
@@ -118,8 +128,23 @@ export default {
         this.loaded = true
         flags.reverse()
         this.flags = flags
+        this.filteredFlags = [...flags];
       }, handleErr.bind(this))
   },
+  watch: {
+    searchTerm: function() {
+      if (this.searchTerm) {
+        this.filteredFlags = this.flags.filter(
+          ({ id, description}) =>
+            id.toString().startsWith(this.searchTerm) ||
+            description.startsWith(this.searchTerm)
```

Comment:
is it possible to make it a substring search instead of just `startsWith`? At least, the search should be as good as cmd+f from the browser. This is equivalent to "description_like" params, so we can easily migrate the search box to server-side search in the future (if it's necessary).

Reply:
@zhouzhuojie  Alright, done.